### ISSUE TITLE
View Post toast is difficult to click

### ISSUE THREAD

**Issue by mkaz** [WordPress/gutenberg#25021]
**Describe the bug**

After updating a post, the View Post toast pops up, far away from the Update button.

This requires moving the mouse to the toast and I invariably always miss the "View Post" link directly and end up clicking the box, by clicking the box it dismisses the toast and does not navigate.

I see various ways to solve, but my most expected behavior is clicking close to the link should navigate towards it, so the whole toast should be clickable. 

Moving it closer or other methods to preview the link, showing permalink, etc... could be alternate solutions.

**Screenshots**

![toast-click](https://user-images.githubusercontent.com/45363/92013983-609fed80-ed03-11ea-9fbb-9174594ddce7.gif)

**Comment by annezazu** [comment]
@mkaz this seems related to this recent issue: https://github.com/WordPress/gutenberg/issues/24915 Can you confirm and close out if so? I agree this definitely needs to be improved!

**Comment by mkaz** [comment]
@annezazu It is quite similar but it depends on the solution, the other issue is advocating for a link near the publish button, if that is done than there would not be a need for the toast.

If the toast is the preferred solution, then my suggestion is to make the whole area near the link clickable.

**Comment by aristath** [comment]
Just my 2c: I'd prefer having the permalink near the publish button. 
Right now the publish button is on the top right, and after clicking it what most authors want to do is to view the post they just published. The toast appears on the bottom left corner, so the opposite of where I had my eyes and it's easy to miss. I was focusing on the publish button so it's outside my line of sight.
That being said, if we do decide to keep the toast, it should be easier to click.
I find myself constantly clicking the preview button because it's hard to find the post's link without going through hoops.

**Comment by annezazu** [comment]
Let's keep this open in that case :)

**Comment by ntsekouras** [comment]
>The toast appears on the bottom left corner, so the opposite of where I had my eyes and it's easy to miss.

While this is true and a bit troubling at times (myself included for the specific use case), the place the toast appears is the same for all toast for any other action as well (such as `copy` action etc..). So I believe it should be consistent and an `active/a bit trained` (that has some experience writing posts) would know that the toasts are there. Of course this is a bit subjective :)

**Comment by aristath** [comment]
I agree that after a while my eyes keep glancing at the borrom-left corner of the screen to see if there's something there I may have missed. But that doesn't make it a good pattern... It's something that I had to get used to (as opposed to something that would have been more natural).

**Comment by youknowriad** [comment]
It's not really clear what's the path forward proposed here? Should we add some design thoughts? 
I know these notices have been discussed at length and there seem to be a decision that they are useful for secondary notices where missing them is "ok".

**Comment by haemeulrich** [comment]
For me as an editor/blogger, it would be helpful to have a shortcut to launch "View Page".

**Comment by mkaz** [comment]
Closing in favor of https://github.com/WordPress/gutenberg/issues/30579
